src/SFA.DAS.AdminService.Web/Infrastructure/RoatpApplicationApiClient.cs

```diff
@@ -221,7 +248,18 @@ private async Task<T> Get<T>(string uri)
             }
         }
 
-        
+        private async Task<string> Get(string uri)
```

Comment:
Is it a good idea to return string from a REST API end point? Usually it would be JSON